build.gradle

```diff
@@ -297,6 +327,7 @@ subprojects { subproj ->
         maven {
             url 'https://airbyte.mycloudrepo.io/public/repositories/airbyte-public-jars/'
         }
+        mavenLocal()
```

Comment:
Is this passing in CI build?

Reply:
@rodireich  No. That's what I'm still trying to debug. If you have any ideas, I'd be very grateful! 🙏

Reply:
@alafanechere it looks like this is the change that I was talking about earlier that will enable connectors to be able to resolve SNAPSHOT jars that are published upstream

Reply:
@alafanechere - This would be great. I put more details on the failure I'm seeing in this thread: https://github.com/airbytehq/airbyte/pull/28687#discussion_r1284776769

Reply:
Passing now. 👍